scripts/amplitude.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.application'
+
+android {
+    buildTypes {
+        def apiKeysProperties = new PropertiesFile(rootProject.file("apikey.properties"), logger)
```

Comment:
add todo about moving to a separate properties.
And did you update the environments?

Reply:
For CI environments nothing was changed, I just updates CI variables